en, zh: add a step for removing Pump nodes (#737)
pingcap/docs-tidb-operator: en/deploy-tidb-binlog.md

### Remove Pump nodes completely

> 1. Before removing Pump nodes, execute `kubectl edit tc ${cluster_name} -n ${namespace}` and **set** `spec.tidb.binlogEnabled` to `false`. After the TiDB Pod is restarted to update the configuration, remove the Pump nodes.

Reviewer: ```suggestion
1. Before removing Pump nodes, execute `kubectl edit tc ${cluster_name} -n ${namespace}` and set `spec.tidb.binlogEnabled` to `false`. After the TiDB Pods are rolling updated, we can remove the Pump nodes.
```

>     * If you directly remove Pump nodes, it might cause TiDB failure because TiDB has no Pump nodes to write into.

Reviewer 2: ```suggestion
    If you directly remove Pump nodes, it might cause TiDB failure because TiDB has no Pump nodes to write into.
```

> 2. Refer to [Scale in Pump](#scale-in-pump) to scale in Pump to `0`.

Reviewer 2: Does it mean `no Pump nodes`?

Author: Yes.
